README.md

```diff
@@ -15,3 +15,11 @@ really care about.
 1. [Estimators](tensorflow/tf-estimators.ipynb) - A guide to the Estimator
    interface.
 
+
+- - - 
+
+## scikit-learn
+
+1. [Model serve](sklearn/gae_serve/README.md) - Serve model with Google App
```

Comment:
I moved this sample under sklearn.  Most definitely the setup itself can be used for models trained with other framework (comment added for this), but I hope to get sklearn users' attention more quickly this way.